### DISCUSSION TITLE
Can upscale latent space image be applied to foreground images before being blended into final image?

### DISCUSSION THREAD

**Discussion by rackSD** [Extraltodeus/multi-subject-render#24] [Q&A]
Generated subjects seem to not have high res fix applied, which causes a great loss of detail in my use case.

**Comment by Extraltodeus** [comment]
If the final blend has a higher resolution that the normal settings (I mean the normal size sliders of the webui) then a highres fix is applied on the final blend automatically.

**Reply by rackSD** [reply]
My bad, meant to say Upscale latent space image when doing hires. fix when blending final image. Can that be applied to the final blend, or is it already in action?

Titled thread wrong, was so used to using upscale latent space image as part of high res. fix that I forgot it's a separate setting.

Edited thread title.
